tutorials/cyber/raw_data_generator/data_generator.yml

```diff
@@ -0,0 +1,45 @@
+name: data_generator
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - boost-cpp=1.66.0=1
+  - bzip2=1.0.6=h14c3975_1002
+  - ca-certificates=2019.3.9=hecc5488_0
+  - certifi=2019.3.9=py36_0
+  - curl=7.62.0=hbc83047_0
+  - gettext=0.19.8.1=hc5be6a0_1002
+  - hdfs3=0.3.0=py36_1000
+  - icu=58.2=hf484d3e_1000
+  - jinja2=2.10=py36_0
+  - krb5=1.14.6=0
```

Comment:
Same question as about hdfs3.. does the notebook need kerberos packages installed?

Reply:
Can you review the rest of the included packages here and remove any that aren't actually needed? It helps keep the size of the generated Docker images down, and avoids possible version conflicts with packages from other notebooks

Reply:
Sure, I will double check how I generated this file

Reply:
I have simplified the file and listed just the few required dependencies